docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following issues.
+
+.Lack of Unique IDs and Aria-Labels on Sections/Categories Hindering Automation
+
+Lack of unique IDs applied to sections and categories hinders migration automation. link:https://issues.redhat.com/browse/MTA-1203[MTA-1203]
+
+.Support downstream disconnected installs
+
+MTA does not support downstream disconnected installs. link:https://issues.redhat.com/browse/MTA-17[MTA-17]
+
+.Jira connection to a removed Wave cannot be removed after it was added to Wave 
+
+Cannot remove a Jira connection that was added to Wave after Wave was removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
```

Comment:
Poor grammar - including missing determiners - **a Wave** without Wave appears to be a verb

```suggestion
Cannot remove a Jira connection that was added to a Wave after the Wave was removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
```

```suggestion
It is not possible to remove a Jira connection that was added to a Wave subsequent to its removal. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
```

```suggestion
It is not possible to remove a connection that was added to a Wave after its removal. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
```